src/renderer/util.ts

```diff
@@ -0,0 +1,22 @@
+import { shell } from "electron";
+
+export function showErrorToast(message: string) {
+	// show toast with error message
+	// @ts-ignore
+	$("body").toast({
+		class: "error",
+		message: `<strong>An unexpected error occured</strong>:<br>${message}`,
+		displayTime: 0,
+		classActions: "top attached",
+		actions: [{
+			text: "Report issue",
+			class: "yellow",
+			click: () => {
+				shell.openExternal("https://github.com/lukechu10/Minecraft-Box-Launcher/issues/new/choose");
+			}
+		}, {
+			text: "Ignore",
+			class: "orange"
+		}]
+	});
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Newline required at end of file but not found.](https://app.codacy.com/manual/lukechu10/Minecraft-Box-Launcher/pullRequest?prid=5146781)